pulseapi/profiles/serializers.py

```diff
@@ -215,6 +215,18 @@ class UserProfileEntriesSerializer(serializers.Serializer):
     returning the number of entries (created, published, and favorited) associated
     with the profile.
     """
+    @staticmethod
+    def get_ordered_queryset(queryset, ordering_param=None, prefix=False):
+        if not ordering_param:
```

Comment:
should this be `if ordering_param is None`? Can it ever something falsey that isn't None?

Reply:
not really and if it does somehow happen to be falsey (like `''`) it should still only return the queryset as is instead of throwing an error and crashing the server when ordering by a falsey value.